### PR TITLE
Fix mandatory field issue

### PR DESCRIPTION
**Pull request by yanjungao718**

### CODE REVIEW

**Comment by wangmingliang-ms** [comment]
We should `Use the **placeholder** to indicate that an input field is optional`
according to [Intellij Platform UI Guildlines](https://jetbrains.github.io/ui/controls/input_field/)
image

**Review by wangmingliang-ms — CHANGES_REQUESTED** [review]
We should `Use the **placeholder** to indicate that an input field is optional`
according to [Intellij Platform UI Guildlines](https://jetbrains.github.io/ui/controls/input_field/)
image

**Comment by yanjungao718** [comment]
remove "optional" text for arisClusterName Field, because it is a HintTextField and already has a default 'optional' value

**Comment by yanjungao718** [comment]
@t-rufang @wangmingliang-ms  please help to review, thank u.

**Review by wangmingliang-ms — DISMISSED** [review]

**Comment by yanjungao718** [comment]
hi @wangmingliang-ms  please approve so I can merge the review, thanks!

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.form [hunk 1]
```diff
@@ -161,16 +161,16 @@
...
                         <properties>
                           <labelFor value="7bd75"/>
-                          <text value="  Server:"/>
+                          <text value="  Server(Required):"/>
                         </properties>
                       </component>
                       <component id="34c8" class="javax.swing.JLabel">
                         <constraints>
-                          <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="4" fill="0" indent="5" use-parent-layout="false"/>
+                          <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="4" fill="0" indent="0" use-parent-layout="false"/>
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.form:169]: Is this indent changes(and following) required?

**Review by wezhang — COMMENTED** [review]
Thank you!

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.form (hunk 1 above)
Comment (yanjungao718) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.form:169]: @wezhang Wei, yes, adding [Required] text on label will make the cell width expanded, change indent can reduce the width.

**Review by t-rufang — DISMISSED** [review]
Thanks, Yanjun! LGTM!

**Review by t-rufang — APPROVED** [review]
Thanks Yanjun!

**Pull request merged**